aws/resource_aws_servicecatalog_portfolio.go

```diff
@@ -0,0 +1,145 @@
+package aws
+
+import (
+	"fmt"
+	"log"
+	"time"
+
+	"github.com/aws/aws-sdk-go/aws"
+	"github.com/aws/aws-sdk-go/aws/awserr"
+	"github.com/aws/aws-sdk-go/service/servicecatalog"
+	"github.com/hashicorp/terraform/helper/schema"
+)
+
+func resourceAwsServiceCatalogPortfolio() *schema.Resource {
+	return &schema.Resource{
+		Create: resourceAwsServiceCatalogPortfolioCreate,
+		Read:   resourceAwsServiceCatalogPortfolioRead,
+		Update: resourceAwsServiceCatalogPortfolioUpdate,
+		Delete: resourceAwsServiceCatalogPortfolioDelete,
+
+		Importer: &schema.ResourceImporter{
+			State: schema.ImportStatePassthrough,
+		},
+
+		Timeouts: &schema.ResourceTimeout{
+			Create: schema.DefaultTimeout(30 * time.Minute),
+			Update: schema.DefaultTimeout(30 * time.Minute),
+			Delete: schema.DefaultTimeout(30 * time.Minute),
+		},
+
+		Schema: map[string]*schema.Schema{
+			"id": {
+				Type:     schema.TypeString,
+				Computed: true,
+				ForceNew: true,
+			},
+			"name": {
+				Type:     schema.TypeString,
+				Required: true,
+				ForceNew: false,
+			},
+			"description": {
+				Type:     schema.TypeString,
+				Optional: true,
```

Comment:
This should be Computed: true, as it may be updated on the console, and we would like to set the value on that :)

(Almost every retrieved attributes can be set as Computed)